pokemon-spring-mvc/src/main/webapp/WEB-INF/jsp/trainer/list.jsp

```diff
@@ -0,0 +1,82 @@
+<%--
+    Document   : Jsp page for listing all trainers.
+    Author     : Michal Mokros 456442
+--%>
+
+<%@ page contentType="text/html;charset=UTF-8" pageEncoding="utf-8" trimDirectiveWhitespaces="false" session="false" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="my" %>
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ taglib prefix="fmt" uri="http://java.sun.com/jsp/jstl/fmt" %>
+<%@ taglib prefix="s" uri="http://www.springframework.org/tags" %>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form" %>
+<%@ taglib prefix="security" uri="http://www.springframework.org/security/tags" %>
+<fmt:setBundle basename="Texts"/>
+<fmt:message var="title" key="trainer.trainers"/>
+<my:pagetemplate title="${title}">
+<jsp:attribute name="body">
+    <h1><fmt:message key="trainer.trainers"/> ${trainer.userName}</h1>
+    <table class="table">
+        <thead>
+        <tr>
+            <th><fmt:message key="trainer.username"/></th>
+            <th><fmt:message key="trainer.name"/></th>
+            <th><fmt:message key="trainer.surname"/></th>
+            <th><fmt:message key="trainer.born"/></th>
+            <th><fmt:message key="trainer.admin"/></th>
+            <th><!--Button Column--></th>
+        </tr>
+        </thead>
+        <tbody>
+        <c:forEach items="${allTrainers}" var="trainer">
+            <tr onclick="window.location='/pa165/trainer/detail/${trainer.id}'" style="cursor: pointer;">
+                <td><c:out value="${trainer.userName}"/></td>
+                <td><c:out value="${trainer.name}"/></td>
+                <td><c:out value="${trainer.surname}"/></td>
+                <td><my:localDate date="${trainer.born}" pattern="dd-MM-yyyy"/></td>

```

Comment:
That thing will become javatime:format (and be locale aware), but it can wait for after the big merges.